TPLsList.cmake

```diff
@@ -58,6 +58,8 @@ TRIBITS_REPOSITORY_DEFINE_TPLS(
   yaml-cpp        "cmake/TPLs/"    EX
   Peano           "cmake/TPLs/"    EX
   CUDA            "${${PROJECT_NAME}_TRIBITS_DIR}/core/std_tpls/"    PT
+  CUBLAS          "cmake/TPLs/"    PT
```

Comment:
Yes, I'm adding new TPLs.  On the other hand, these are trivial to enable when CUDA is enabled: just set `TPL_ENABLE_CUBLAS=ON` etc.  I will work on automating that.